apps/editor.planx.uk/src/ui/public/ApplicationSummary.tsx

```diff
@@ -0,0 +1,64 @@
+import Box from "@mui/material/Box";
+import { SummaryListTable } from "@planx/components/shared/Preview/SummaryList";
+import { objectWithoutNullishValues } from "lib/objectHelpers";
+import { type Store,useStore } from "pages/FlowEditor/lib/store";
+import { Fragment } from "react";
+import React from "react";
+
+
+// TODO - Retire in favour of ODP Schema application type descriptions or fallback to flowName
+function getWorkStatus(passport: Store.Passport): string | undefined {
+  switch (passport?.data?.["application.type"]?.toString()) {
+    case "ldc.existing":
+      return "existing";
+    case "ldc.proposed":
+      return "proposed";
+  }
+}
```

Comment:
nit: I think we _can_ replace this now using planx-core's `getValidSchemaValues("ApplicationType")` ?? 

Or do we have another exported helper which returns both value _and_ description? Not a blocker, but a nice cleanup if we can!